examples/app.py

```diff
@@ -24,7 +24,12 @@
 
 Session(app)
 
-auth_manager = spotipy.oauth2.SpotifyOAuth()
+CACHE = ".cache-" + "test"
```

Comment:
This will look like it works but in fact if user B signs in right after user A, user B will see A's data.

https://github.com/plamere/spotipy/pull/539 should fix multi-users usage, so let's not worry about that in the current PR

For now you can remove this line